src/store/reducers/drinkTypes.js

```diff
@@ -1,5 +1,29 @@
-import { initialState } from '../data/drinkTypes';
+import * as actions from '../actions/actionTypes';
 
-const drinkTypesReducer = (state = initialState, action) => state;
+const setTypes = (state, action) => {
+  
+  const results = action.drinks.map((type, key) => {
```

Comment:
dont you thing this is pretty similar to the one that is being used in alcohol.js? DRY